docs/en/stack/ml/df-analytics/ml-dfa-phases.asciidoc

```diff
@@ -0,0 +1,72 @@
+[role="xpack"]
+[[ml-dfa-phases]]
+=== How a {dfanalytics-job} works
+
+This page provides high-level background information on how {dfanalytics-jobs} 
```

Comment:
> This page...

In my opinion, this introductory text isn't required.  It's a short page and the title is quite clear, so I don't think people will be confused when they land here.